<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popupMessageDialog</class>
 <widget class="QDialog" name="popupMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="popupMessageDialogStackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QWidget" name="popupMessageDialogErrorPage">
    <property name="styleSheet">
     <string notr="true">#popupMessageDialogErrorPage
{
	background-image: url(:/Resources/Resources/popupError.png);
}
</string>
    </property>
    <widget class="QLabel" name="popupMessageDialogErrorLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>התרחשה שגיאה</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="popupMessageDialogInfoPage">
    <property name="styleSheet">
     <string notr="true">#popupMessageDialogInfoPage
{
	background-image: url(:/Resources/Resources/popupInfo.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="popupMessageDialogCSVPage">
    <property name="styleSheet">
     <string notr="true">#popupMessageDialogCSVPage
{
	background-image: url(:/Resources/Resources/popupInfo.png);
}</string>
    </property>
    <widget class="QLabel" name="popupMessageDialogCSVHeader">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>סיכום הגירת נתונים למסד</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVLabel1">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>מתוך</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVLinesLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVLabel2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>רשומות, </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVErrorsLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVLabel3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>הועתקו בהצלחה</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="popupMessageDialogCSVLabel4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>לחץ על &quot;פרטים&quot; לקבלת נתונים נוספים</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="popupMessageDialogGoodPage">
    <property name="styleSheet">
     <string notr="true">#popupMessageDialogGoodPage
{
background-image: url(:/Resources/Resources/popupGood.png);
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>הפעולה הצליחה</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="popupMessageDialogOKBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#popupMessageDialogOKBtn
{	
	background-image: url(:/Resources/Resources/OKBtn.png);
	border: 0px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="popupMessageDialogDetailBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>130</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#popupMessageDialogDetailBtn
{	
	background-image: url(:/Resources/Resources/detalesBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="popupMessageDialogTextEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>430</width>
     <height>220</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
